src/components/ModulePanel.vue

```diff
@@ -68,6 +70,15 @@ export default defineComponent({
   setup() {
     const selectedModuleIndex = ref(0);
 
+    const toolStore = useToolStore();
+    watch(
+      () => toolStore.currentTool,
+      (newTool) => {
+        if ([Tools.Rectangle, Tools.Ruler].includes(newTool))
```

Comment:
I would recommend making this array a module-level constant for visibility purposes.